### PR TITLE
feat: highlight selected account in menu

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

I have added a `selected` prop (as active is already in use because of Headless UI) to highlight the selected button in menu.

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes (If any)
Screenshot from 2022-06-03 23-07-14

### Note

Do we have to update the stories to include the new selected prop? 
(`/src/app/components/Menu/Menu.stories.tsx`)

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

File: src/app/components/AccountMenu/index.tsx [hunk 1]
```diff
@@ -83,6 +83,7 @@ function AccountMenu({ title, subtitle, showOptions = true }: Props) {
                   selectAccount(accountId);
                 }}
                 disabled={loading}
+                selected={accountId === auth.account?.id}
```
Comment (escapedcat) [inline src/app/components/AccountMenu/index.tsx:86]: Would you mind extending this test with the new `selected` prop?:  
https://github.com/getAlby/lightning-browser-extension/blob/ed6bb1aa3ae39ed674abc8f686fb3f64ef867541/src/app/components/AccountMenu/index.test.tsx  

i.e. add a test "Current account is being highlighted"

**Review by escapedcat — CHANGES_REQUESTED** [review]

**Comment by escapedcat** [comment]
In general I'm always a bit conflicted using `bold` for highlights because the text will be wider. If a text is already long this might break things. Not sure if this can be the case here though. Just a thing to keep in mind and maybe test it.

nit: Not sure if this a bugfix. It's more like a new feature I think.

Thanks for your work!

**Comment by im-adithya** [comment]
> Would you mind extending this test with the new selected prop?:

Done! :D

> nit: Not sure if this a bugfix. It's more like a new feature I think.

Changed the commit message and title!

**Comment by im-adithya** [comment]
Also we should wait for #992 to be merged so that I can change it here as well!
(i.e. `fireEvent` to `userEvent`)

**Comment by im-adithya** [comment]
> so that I can change it here as well! (i.e. `fireEvent` to `userEvent`)

Done!

**Review by bumi — APPROVED** [review]
thanks! looks good to me. 
I think the bold can easily be changed when we maybe chat about the design with @dvoroneca

**Review by escapedcat — APPROVED** [review]

**Comment by escapedcat** [comment]
@im-adithya fyi, @dylancom changed it a bit here to avoid changing "all" menus:  #1001

**Pull request merged**